coresdk/src/coresdk/types.h

```diff
@@ -43,6 +43,14 @@ enum font_style
     UNDERLINE_FONT   = 4
 };
 
+/// Provides an overload that allows combining font styles with the bit or 
+/// operator.
+/// Source: http://stackoverflow.com/a/1448478
+inline font_style operator|(font_style a, font_style b)
```

Comment:
Good idea -- but mapping this to other languages may be a challenge! @alexcu thoughts?